src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -57,7 +55,6 @@
 
   <PropertyGroup Condition="'$(TestWithCore)' != 'true' And '$(TestAppX)' != 'true'">
     <XunitExecutable Condition="'$(XunitExecutable)' == ''">xunit.console.exe</XunitExecutable>
```

Comment:
I don't know if we have any real cases but we should try and verify that this works for non-core debugging.